internal/plugin.go

```diff
@@ -1237,6 +1240,26 @@ func assertValidPlacement(
 		}
 	}
 
+	if len(placement.ClusterSelectors) > 0 && len(placement.ClusterSelector) > 0 {
+		return fmt.Errorf("cannot use both clusterSelector and clusterSelectors in placement config " +
```

Comment:
Could you use the `path` method argument in the error message so that the user knows where the invalid selector is?